vue/.eslintrc.js

```diff
@@ -12,7 +12,8 @@ module.exports = {
     'portValue': true,
     'hostedByLogoUrl': true,
     'hostedByText': true,
-    'titleText': true
+    'titleText': true,
```

Comment:
https://eslint.org/docs/rules/comma-dangle

Reply:
i will fix those with the eslint ticket